src/pages/About/index.js

```diff
@@ -15,6 +18,23 @@ const useStyles = makeStyles((theme) => ({
     },
     subHeading: {
         padding: `${theme.spacing(2)}px 0 ${theme.spacing(2)}px 0`
+    },
+    link: {
```

Comment:
I've just copied these stylings from the pathway page (i think...)

where would you put these `link` and `listItem` styles to avoid duplication @Caitlin-cooling ?

Reply:
Yeah this is something that I have been thinking about recently. I don't know the best way to go about it in material UI. Looks like you can make global overrides for their components, but the links are not material UI components.

My gut says that we want a global styles file but I'm going to need to think about it a bit more. When do you want this merged by?

Reply:
I'd quite like to get it in for the demo - we could probably create a refactoring ticket to de-dupe any styling blocks common to multiple pages?

Reply:
Ah sorry didn't see this! I do actually think it's worth handling as a larger tech debt piece have made a ticket https://devservices.jira.com/browse/LFH-134